lib/client.js

```diff
@@ -88,7 +88,7 @@ exports.boot = function boot(options, callback) {
     // render the factory on the client
     // doing this, sets up the event
     // listeners and stuff aka mounting views.
-    React.render(viewFactory(props), mountNode);
+    ReactDOM.render(viewFactory(props), _document);
```

Comment:
Are we no longer allowing the node to render to via the options.mountNode? If we are no longer using mountNode remove it at line 62.